api/src/main/java/org/pmiops/workbench/api/ClusterController.java

```diff
@@ -116,7 +123,51 @@
   }
 
   @Override
-  public ResponseEntity<ClusterListResponse> listClusters(
+  @AuthorityRequired(Authority.SECURITY_ADMIN)
+  public ResponseEntity<List<ListClusterResponse>> deleteClustersInProject(String billingProjectId) {
+    if (billingProjectId == null) {
+      throw new BadRequestException("Must specify billing project");
+    }
+    clusterAuditor.fireDeleteClustersInProject(billingProjectId);
```

Comment:
I'd be inclined to move this towards the bottom of this method, so the audit log is only sent after the Firecloud interactions are completed. Just checking how Jay set this up for other controllers, there seems to be a pretty clear pattern of "do the work first, then fire an audit log".